### ISSUE TITLE
Failed to upload an attachment to a card

### ISSUE THREAD

**Issue by hoffrichterm** [stefan-niedermann/nextcloud-deck#1102]
**Describe the bug**
When I try to attach a Photo in the Decks app  an Exception is thrown

Open a card
Click on the Upload Symbol
Choose a Photo
Click on Upload

**Expected behavior**
The file should be attached to the card

**Screenshots**

![Screenshot_20210910-092333](https://user-images.githubusercontent.com/50859985/132816222-9937ddcd-cbff-4e3f-8a22-e6fed012ad77.png)
![Screenshot_20210910-092316](https://user-images.githubusercontent.com/50859985/132816228-073b7abb-3308-4065-8dc5-06916773fe1f.png)

**Versions**

 - Nextcloud: 22.1.1
 - Nextcloud Deck: 1.5.1
 - Nextcloud Android: 3.17.0
 - Nextcloud Android Deck: 1.19.3

**Smartphone (please complete the following information):**
- Device:  Google Pixel 4
- Android-Version:  11
- App-Store:
  - [x] Google Play Store
  - [ ] Google Play Store (Beta channel)
  - [ ] F-Droid
  - [ ] Huawei AppGallery

**Stacktrace**

```
App Version: 1.19.3
App Version Code: 1019003
Server App Version: 1.5.1
App Flavor: play

Files App Version Code: 30170090

---

OS Version: 4.14.212-g3fa89efea9d1-ab7460272(7474174)
OS API Level: 30
Device: flame
Manufacturer: Google
Model (and Product): Pixel 4 (flame)

---

java.io.IOException: Failed to create cacheFile
	at it.niedermann.nextcloud.deck.util.FilesUtil.getTempCacheFile(FilesUtil.java:73)
	at it.niedermann.nextcloud.deck.util.FilesUtil.copyContentUriToTempFile(FilesUtil.java:36)
	at it.niedermann.nextcloud.deck.ui.card.attachments.CardAttachmentsFragment.lambda$uploadNewAttachmentFromUri$11$CardAttachmentsFragment(CardAttachmentsFragment.java:410)
	at it.niedermann.nextcloud.deck.ui.card.attachments.CardAttachmentsFragment$$ExternalSyntheticLambda1.run(Unknown Source:6)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)

```

**Comment by stefan-niedermann** [comment]
Hi @hoffrichterm 👋 

1. Does this happen always when you try to upload an attachment, sometimes or only once?
2. Can you please check how much free storage do you have left on your phone?

**Comment by hoffrichterm** [comment]
Hi @stefan-niedermann 

It happens always. 
And here is a screenshot:
![Screenshot_20210910-102109](https://user-images.githubusercontent.com/50859985/132823810-f9059c0b-67bc-4ba5-8d95-003b5d7508bc.png)

**Comment by stefan-niedermann** [comment]
Hm, that's quite weird :thinking: Is only this picture affected or literally any image, file and contact you try to attach? How did you manage to upload the other attachments one can see in the screenshot? Via the Web UI?

**Comment by stefan-niedermann** [comment]
Allright, we think we found the reason for this problem and we fixed it in the future `1.19.4` release. Let us know in case it occurs again with `1.19.4` or later.
Thanks for the detailed bug report!

Also for the sake of completeness: Marking as duplicate of #1075
